src/content/docs/logs/ui-data/query-time-parsing.mdx

```diff
@@ -0,0 +1,412 @@
+        * `appId`
+        * `endTimestamp`
+        * `eventType`
+        * `logs.accountId`
+        * `logs.hashId`
+        * `messageId`
+        * `message_id`
+        * `metricName`
+        * `metricTimesliceName`
+        * `newrelic.timeslice.value`
+        * `timestamp`
+        * `timestampSeconds`
+        * `timestampMillis`
+        * Can't start with `nr.` or `newrelic`
+    * You can't name an extracted value with the same name of the attribute that is being parsed. For example, if you are creating a query time parsing rule for the message attribute, an extracted value cannot be named message.
+        * Extracted value names must be unique across all query time parsing rules.
+        * If an extracted value is given the same name as an existing attribute in your logs, the results in the log table for that column will show data from ingest and data from query time parsing rules. 
+    </Collapser>
+</CollapserGroup>
+
+    </Step>
+    <Step>
+        #### Finish creating your query time parsing rule [#finish-rule]
+
+        After you've selected values, complete the following:
+
+        1. In the editor, review the preview of the table showing how your rule will be applied to the log you selected. 
+
+            <img
+                title="Screenshot showing how the editor shows a preview of the rule"
+                alt="Screenshot showing how the editor shows a preview of the rule"
+                src={logsEditorShowsParsedDataPreview}
+            />
+
+        2. If you're interested in the NRQL function used in the query to get your logs, click on **Query**. 
+
+            <img
+                title="Screenshot showing the query behind the parsing rule"
+                alt="Screenshot showng the query behind the parsing rule"
+                src={logsNrqlBehindQueryRule}
+            />
+
+            <figcaption>
+            The pattern string, which is used to find and extract value(s) from the selected attribute, is updated whenever a value is extracted.
+            </figcaption>
+
+        3. If you need to rename any variables before you create your rule, click on the name, provide a new name, and the click **Save**. You can also delete variables by clicking on the variable you wish to delete and clicking **Delete**.
+            <img
+                title="Screenshot showing where you can delete values"
+                alt="Screenshot showing where you can delete values"
+                src={logsDeleteQueryTimeRule}
+            />
+            <figcaption>
+            You can edit or delete variables by clicking on them.
+            </figcaption>
+        4. After you've finished reviewing and editing your variable names, click **Create rule** to finish creating your rule. 
+
+        5. If you started creating a query time parsing rule from the **Log details** view, you need to close that view to see your results in the log table.
```

Comment:
Is this a step or is this just a tip or a note?

Reply:
@homelessbirds you're right that this wasn't really a step. It's more of a note. I moved this into a paragraph outside the procedure. What do you think?